src/oauth-service.ts

```diff
@@ -88,7 +88,8 @@ export class OAuthService implements IOAuthService {
 
     public logout = (): void => {
         var redirectUrl = `${this.config.logoutUrl}?` +
-            `${this.config.logoutRedirectParameterName}=${encodeURIComponent(this.config.redirectUri)}`;
+            `${this.config.logoutRedirectParameterName}=${encodeURIComponent(this.config.redirectUri)}&` +
+            `id_token_hint=${encodeURIComponent(this.oAuthTokenService.getIdToken())}`;

```

Comment:
This commit need small changes:
- Add 'id_token_hint' hard-coded string to the OAuth config object (defaults: IOAuthConfig) with default value empty value '' for backword compatibility
- Change the line above to use config value for the parameter name rather then the string
- Separate adding this parameter to redirectUrl in the if statement only when to add it only when the parameter was set up via configuration, similar code as for scope in login method
- Add new parameter to the custom typings file to reflect new configuration parameter - adjust interface IOAuthConfig